src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.revature.domain.Person;
+import com.revature.repositories.PersonRepository;
+
+@Service
+@Transactional(readOnly=false, isolation=Isolation.READ_COMMITTED)
+public class PersonLogicImpl implements PersonLogic {
+
+	@Autowired
+	private PersonRepository personRepository;
+	
+	@Override
+	public List<Person> getPersonByFirstName(String firstName) {
+		
+		List<Person> persons = personRepository.findByFirstName(firstName);
+		
+		return persons;
+	}
+
+	@Override
+	public List<Person> getPersonByLastName(String lastName) {
+		
+		List<Person> persons = personRepository.findByLastName(lastName);
+		
+		return persons;
+	}
+	
+public static void main(String[] args) {
+	PersonLogic pLog = new PersonLogicImpl();
+	Person pers = (Person) pLog.getPersonByFirstName("Efren");
+	System.out.println(pLog);
+	System.out.println(pers);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)